frame/core-fellowship/Cargo.toml

```diff
@@ -25,7 +25,7 @@ sp-io = { version = "7.0.0", default-features = false, path = "../../primitives/
 sp-runtime = { version = "7.0.0", default-features = false, path = "../../primitives/runtime" }
 sp-std = { version = "5.0.0", default-features = false, path = "../../primitives/std" }
 
-[dev-dependencies]
+# for runtime-benchmarks
 pallet-ranked-collective = { version = "4.0.0-dev", default-features = false, path = "../ranked-collective" }
 pallet-salary = { version = "4.0.0-dev", default-features = false, path = "../salary" }
```

Comment:
Oh thanks, good point!

Reply:
According to the same docs page,

> The "package-name/feature-name" syntax will also enable package-name if it is an optional dependency.

So it shouldn't be strictly neccessary, but I could still add `dep:pallet-salary` etc to the runtime-benchmarks matrix simply to avoid exposing a `pallet-salary` feature flag.